### PR TITLE
Regen Ledger v4.1 Upgrade Proposal

### PR DESCRIPTION
**Pull request by clevinson**
## Summary

This pull request includes the proposal json and long-form proposal for the regen ledger v4.1 upgrade.

See rendered preview [here](https://github.com/regen-network/governance/blob/clev/regen-4.1/proposals/2022-09-regen-ledger-v4.1-upgrade/README.md).

### CODE REVIEW

File: proposals/2022-09-regen-ledger-v4.1-upgrade/proposal.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Regen Ledger v4.0 Upgrade",
+  "description": "This is a software upgrade proposal for the upgrade to Regen Ledger v4.1. If passed, this proposal would commit Regen Mainnet to halting the application binary for Regen Ledger v4.0 at approximately 17:00 UTC on October 6th at block height 7466120 and requiring nodes to restart with the application binary for Regen Ledger v4.1. \n\nMore details can be found in the long form proposal: https://github.com/regen-network/governance/tree/main/proposals/2022-09-regen-ledger-v4.1-upgrade",
+  "upgrade-height": 7466120,
```
Comment (clevinson) [inline proposals/2022-09-regen-ledger-v4.1-upgrade/proposal.json]: https://www.mintscan.io/regen/blocks/7466120

File: proposals/2022-09-regen-ledger-v4.1-upgrade/proposal.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "upgrade-info": {
+    "binaries": {
+      "linux/amd64": "https://github.com/regen-network/regen-ledger/releases/download/XXXXXXXX",
```
Comment (clevinson) [inline proposals/2022-09-regen-ledger-v4.1-upgrade/proposal.json]: todo: Add binary URLs after final release is published.

File: proposals/2022-09-regen-ledger-v4.1-upgrade/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Regen Ledger v4.1 Upgrade
+
+This is a software upgrade proposal for the upgrade to Regen Ledger v4.1. If passed, this proposal would commit Regen Mainnet to halting the application binary for Regen Ledger v4.0 at approximately 16:00 UTC on October 7th and starting the application binary for Regen Ledger v4.1.
```
Comment (anilcse) [inline proposals/2022-09-regen-ledger-v4.1-upgrade/README.md]: Let's mention upgrade height

File: proposals/2022-09-regen-ledger-v4.1-upgrade/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+A fix for this issue has been implemented in Regen v4.1. It will, at the scheduled upgrade height, update Tendermint's validator set to the correct active set (as managed by the application). In other words, all validators who should have been jailed since the upgrade to v4.0 will be immediately removed from Tendermint's validator set, and any new validators waiting to be admitted into the active set will be immediately added to tendermint's block-producing validator set.
+
+No user funds were lost in this process. However, it is important to point out that validators who were supposed to be jailed were not actually jailed, but remained in Tendermint's active set and continued to receive rewards (delegators who staked with these validators also still received rewards) as these validator continued to produce blocks. Slashing still occurred exactly as expected and these validators were still slashed for their behavior. We invite the validator community to independently evaluate whether it is necessary to redistribute rewards that may have been affected by this bug. If necessary, the core devs will plan a second upgrade that redistributes rewards accordingly after this upgrade.
```
Comment (anilcse) [inline proposals/2022-09-regen-ledger-v4.1-upgrade/README.md]: Should we specifically mention about deciding distribution of these rewards are independent of this proposal. I would advise to remove these details from proposal if possible, as it is confusing more than helping. We can coordinate this in validator channel if required.

**Review by anilcse — COMMENTED** [review]
LGTM, just small nits

File: proposals/2022-09-regen-ledger-v4.1-upgrade/README.md (hunk 3 above)
Comment (clevinson) [reply proposals/2022-09-regen-ledger-v4.1-upgrade/README.md]: Done

File: proposals/2022-09-regen-ledger-v4.1-upgrade/README.md (hunk 4 above)
Comment (clevinson) [reply proposals/2022-09-regen-ledger-v4.1-upgrade/README.md]: Updated. Thanks for the suggestion

**Review by technicallyty — APPROVED** [review]
LGTM

**Review by aaronc — APPROVED** [review]

**Pull request merged**